docs/appendix/track.md

```diff
@@ -12,4 +12,24 @@
 
 > When you "track" a cellphone, you monitor [the starting] location ... and follow it wherever it goes...
 
-One of the most-often used traces is to find the origins of health problems like mad cow disease and tainted drugs and food. In Value Flows terms, that would start with the product that caused a health problem, and tracing back along the chain of resource flows to the source or sources of the product.
\ No newline at end of file
+One of the most-often used traces is to find the origins of health problems like mad cow disease and tainted drugs and food. In Value Flows terms, that would start with the product (economic resource) that caused a health problem, and trace back along the chain of resource flows to the source or sources of the product.  Then it might be necessary to also track forwards from the source(s) to find everything else that might include that source(s).
+
+Here is the logic for tracking and tracing.
+
+* For an EconomicResource:
+    * after: EconomicEvents that are inputs to Processes or Transfers
+    * before: EconomicEvents that are outputs from Processes or Transfers
```

Comment:
I think we may wan to clarify "EconomicEvents affecting it that ...."

Reply:
good idea